server/routes/userImages.js

```diff
@@ -4,7 +4,7 @@ const express = require('express');
 const router = express.Router();
 const User = require('../database/models/User');
 const isLoggedInGuard = require('../middleware/isLoggedInGuard');
-const ownershipGuard = require('../middleware/ownershipGuard');
+const itemOwnerGuard = require('../middleware/itemOwnerGuard');
```

Comment:
don't need the itemOwnerGuard on this route (will stop people from changing their profile pic).

Looking at this route again, I will need to update the route itself so that put doesn't take a /:userId param... that way the route will still be protected (not a guard issue)